base/system/userinit/livecd.c

```diff
@@ -33,14 +33,18 @@ InitLogo(PIMGINFO pImgInfo, HWND hwndDlg)
     hLogo = (HBITMAP)LoadImageW(hInstance, MAKEINTRESOURCEW(IDB_ROSLOGO), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
     hMask = (HBITMAP)LoadImageW(hInstance, MAKEINTRESOURCEW(IDB_ROSMASK), IMAGE_BITMAP, 0, 0, LR_DEFAULTCOLOR);
 
-    if (hLogo != NULL && hMask != NULL)
+    if (hLogo != NULL && hMask != NULL && hDCLogo != NULL && hDCMask != NULL)
     {
-        GetObject(hLogo, sizeof(logoBitmap), &logoBitmap);
-        GetObject(hMask, sizeof(maskBitmap), &maskBitmap);
-
-        if (logoBitmap.bmHeight != maskBitmap.bmHeight || logoBitmap.bmWidth != maskBitmap.bmWidth)
+        GetObject(hLogo, sizeof(BITMAP), &logoBitmap);
+        GetObject(hMask, sizeof(BITMAP), &maskBitmap);
```

Comment:
Unwanted changes.

Reply:
Idea was to align SYSDM and LiveCD inituser for consistency and later maintenance.
Would you prefer 
-sizeof(BITMAP) in both
-sizeof(name_of_variable) in both
-keep as it is with different design

Reply:
OK